debugtools/DDR_VM/src/com/ibm/j9ddr/corereaders/macho/ThreadCommand.java

```diff
@@ -90,9 +90,13 @@ public ThreadState(int flavor, int size, byte state[], ByteOrder endian)
 			registers = new TreeMap<>();
 			switch (flavor) {
 				case x86_THREAD_STATE:
+				case x86_THREAD_STATE32
```

Comment:
Do `x86_THREAD_STATE32` and `x86_EXCEPTION_STATE32` actually occur in 64-bit dumps?
If so, is the format the same (a sequence of mostly 64-bit register values)? I expect not.

Reply:
Yes you're correct, thread state for 64-bit would have double register size and r8-15. For exception state, it looks like the faultaddr is also 64 bits instead of 32.